internal/tsc_wrapped/compiler_host.ts

```diff
@@ -7,6 +7,7 @@ import {FileLoader} from './file_cache';
 import * as perfTrace from './perf_trace';
 import {BazelOptions} from './tsconfig';
 import {DEBUG, debug} from './worker';
+import {relative, join} from './path';
```

Comment:
how about `import * as path from './path'` so you don't need other changes below